network/src/p2p/handler.rs

```diff
@@ -339,6 +339,15 @@ impl Handler {
             max_peers,
         })
     }
+
+    pub fn get_port(&self) -> IoHandlerResult<u16> {
```

Comment:
Please make `get_port` return `u16` and `get_peer_count` return `usize`.
There is no reason to return `Result`.